tests/common.py

```diff
@@ -1,5 +1,7 @@
+import os
+
 MASTER_KEY = 'masterKey'
-BASE_URL = 'http://127.0.0.1:7700'
+BASE_URL = os.getenv('MEILISEARCH_HOST') if os.getenv('MEILISEARCH_HOST') else 'http://127.0.0.1:7700'
```

Comment:
Something like this, works?

```suggestion
BASE_URL = os.getenv('MEILISEARCH_HOST') || 'http://127.0.0.1:7700'
```

Reply:
Nop:
```
E       BASE_URL = os.getenv('MEILISEARCH_HOST') || 'http://127.0.0.1:7700'
E                                                 ^
E   SyntaxError: invalid syntax
```